packages/appstore/src/components/multi-action-button-group/__test__/multi-action-button-group.spec.tsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import { screen, render } from '@testing-library/react';
+import { StoreProvider, mockStore } from '@deriv/stores';
+import { createBrowserHistory } from 'history';
+import userEvent from '@testing-library/user-event';
+import { Router } from 'react-router';
+import MultiActionButtonGroup from '../multi-action-button-group';
+import { routes } from '@deriv/shared';
+
+let mock_props = {
+    link_to: routes.trade,
+    onAction: jest.fn(),
+    is_buttons_disabled: false,
+    is_real: false,
+};
+
+const renderTestComponent = (props: any) => {
+    const history = createBrowserHistory();
+    const mock = mockStore({
+        modules: {
+            cfd: {
+                dxtrade_tokens: '',
+            },
+        },
+        traders_hub: {
+            is_demo: true,
+        },
+    });
+    return render(
+        <StoreProvider store={mock}>
+            <Router history={history}>
+                <MultiActionButtonGroup {...props} />
+            </Router>
+        </StoreProvider>
+    );
+};
```

Comment:
Dont think this is advisable. 
cc: @farzin-deriv

Reply:
Already made the changes 😃 👍🏼